### PR TITLE
Java quick start

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by chbatey** [comment]
This is now working for sbt, maven and gradle. Requires the latest akka-grpc so suggesting we release a snaphot or milestone release so we can add CI to these samples

File: src/main/g8/src/main/java/com/example/helloworld/GreeterServer.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    bound.thenAccept(binding ->
+        System.out.println("gRPC server bound to: " + binding.localAddress())
+    );
+
+    return bound;
+  }
+  // #server
+
+
+  // FIXME this will be replaced by a more convenient utility, see https://github.com/akka/akka-grpc/issues/89
```
Comment (raboof) [inline src/main/g8/src/main/java/com/example/helloworld/GreeterServer.java]: hmm not sure we figured out yet what to do here, but we can leave that for another PR

File: docs/src/main/paradox/index.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ As you saw in the console output, the example outputs several greetings. Let’s
 First, the `GreeterServer` main class creates an `akka.actor.ActorSystem`, a container in which Actors, Akka Streams and Akka HTTP run. Next, it defines a function from `HttpRequest` to `Future[HttpResponse]` using the `GreeterServiceImpl`. This function
 handles gRPC requests in the HTTP/2 with TLS server that is bound to port 8080 in this example.
 
-@@snip [GreeterServer.scala]($g8src$/scala/com/example/helloworld/GreeterServer.scala) { #import #server }
+@@snip [GreeterServer.java]($g8src$/java/com/example/helloworld/GreeterServer.java) { #import #server }
 
 `GreeterServiceImpl` is our implementation of the gRPC service, but first we must define the interface of the service
 in the protobuf file `src/main/protobuf/helloworld.proto`:
```
Comment (raboof) [inline docs/src/main/paradox/index.md]: This was moved to `src/main/proto/helloworld.proto`

**Comment by ktoso** [comment]
Triggering travis I hope
